exchanges/bybit/bybit_test.go

```diff
@@ -85,3502 +74,2995 @@ func TestGetKlines(t *testing.T) {
-	_, err := b.BatchCancelOrderByIDs(context.Background(), []string{"889208273689997824", "889208273689997825"})
+	_, err := b.GetRiskLimit(context.Background(), "linear", linearTradablePair.String())
 	if err != nil {
-		t.Fatal(err)
+		t.Error(err)
 	}
-}
-
-func TestListOpenOrders(t *testing.T) {
-	t.Parallel()
-	sharedtestvalues.SkipTestIfCredentialsUnset(t, b)
-
-	_, err := b.ListOpenOrders(context.Background(), "BTCUSDT", "", 0)
+	_, err = b.GetRiskLimit(context.Background(), "inverse", inverseTradablePair.String())
 	if err != nil {
-		t.Fatal(err)
+		t.Error(err)
 	}
-}
-
-func TestGetPastOrders(t *testing.T) {
-	t.Parallel()
-	sharedtestvalues.SkipTestIfCredentialsUnset(t, b)
-
-	_, err := b.GetPastOrders(context.Background(), "BTCUSDT", "", 0, time.Now().Add(-time.Hour), time.Now())
-	if err != nil {
-		t.Fatal(err)
+	_, err = b.GetRiskLimit(context.Background(), "option", optionsTradablePair.String())
+	if !errors.Is(err, errInvalidCategory) {
+		t.Error(err)
+	}
+	_, err = b.GetRiskLimit(context.Background(), "spot", spotTradablePair.String())
+	if !errors.Is(err, errInvalidCategory) {
+		t.Error(err)
 	}
 }
 
-func TestGetTradeHistory(t *testing.T) {
+// test cases for Wrapper
+func TestUpdateTicker(t *testing.T) {
 	t.Parallel()
-	sharedtestvalues.SkipTestIfCredentialsUnset(t, b)
-
-	_, err := b.GetTradeHistory(context.Background(), 0, "", "", "", "", time.Now().Add(-time.Hour), time.Now())
+	_, err := b.UpdateTicker(context.Background(), spotTradablePair, asset.Spot)
 	if err != nil {
-		t.Fatal(err)
+		t.Error(err)
 	}
-}
-
-func TestGetWalletBalance(t *testing.T) {
-	t.Parallel()
-	sharedtestvalues.SkipTestIfCredentialsUnset(t, b)
-
-	_, err := b.GetWalletBalance(context.Background())
+	_, err = b.UpdateTicker(context.Background(), linearTradablePair, asset.USDTMarginedFutures)
 	if err != nil {
-		t.Fatal(err)
+		t.Error(err)
 	}
-}
-
-func TestGetSpotServerTime(t *testing.T) {
-	t.Parallel()
-	_, err := b.GetSpotServerTime(context.Background())
+	_, err = b.UpdateTicker(context.Background(), inverseTradablePair, asset.CoinMarginedFutures)
 	if err != nil {
-		t.Fatal(err)
+		t.Error(err)
 	}
-}
-
-func TestGetDepositAddressForCurrency(t *testing.T) {
-	t.Parallel()
-	sharedtestvalues.SkipTestIfCredentialsUnset(t, b)
-
-	_, err := b.GetDepositAddressForCurrency(context.Background(), currency.BTC.String())
+	_, err = b.UpdateTicker(context.Background(), optionsTradablePair, asset.Options)
 	if err != nil {
-		t.Fatal(err)
+		t.Error(err)
 	}
 }
 
-func TestWithdrawFund(t *testing.T) {
+func TestUpdateOrderbook(t *testing.T) {
 	t.Parallel()
-	sharedtestvalues.SkipTestIfCredentialsUnset(t, b)
-
-	_, err := b.WithdrawFund(context.Background(), currency.ETH.String(), currency.ETH.String(), "0xEA13A385BcB74e631AAF1B424d7a01c61bF27Fe0", "", "10")
-	if err != nil && err.Error() != "Withdraw address chain or destination tag are not equal" {
-		t.Fatal(err)
+	_, err := b.UpdateOrderbook(context.Background(), spotTradablePair, asset.Spot)
```

Comment:
I'm noting that you're not testing USDCMarginedFutures in any cases, could you please add them?